plugins/modules/homebrew_services.py

```diff
@@ -0,0 +1,273 @@
+    package = module.params["name"]  # type: ignore
+    if not HomebrewValidate.valid_package(package):
+        module.fail_json(msg="Invalid package name: {0}".format(package))
+
+    state = module.params["state"]  # type: ignore
+    if state not in ["present", "absent", "restarted"]:
+        module.fail_json(msg="Invalid state: {0}".format(state))
+
+    brew_path = parse_brew_path(module)
+
+    return HomebrewServiceArgs(name=package, state=state, brew_path=brew_path)
+
+
+def start_service(args, module):
+    # type: (HomebrewServiceArgs, AnsibleModule) -> None
+    """Start the requested brew service if it is not already running."""
+    state = _brew_service_state(args, module)
+    if state.running:
+        # Nothing to do, return early.
+        _exit_with_state(args, module, changed=False, message="Service already running")
+
+    if module.check_mode:
+        _exit_with_state(args, module, changed=True, message="Service would be started")
+
+    start_cmd = [args.brew_path, "services", "start", args.name]
+    rc, stdout, stderr = module.run_command(start_cmd, check_rc=True)
+
+    _exit_with_state(args, module, changed=True)
+
+
+def stop_service(args, module):
+    # type: (HomebrewServiceArgs, AnsibleModule) -> None
+    """Stop the requested brew service if it is running."""
+    state = _brew_service_state(args, module)
+    if not state.running:
+        # Nothing to do, return early.
+        _exit_with_state(args, module, changed=False, message="Service already stopped")
+
+    if module.check_mode:
+        _exit_with_state(args, module, changed=True, message="Service would be stopped")
+
+    stop_cmd = [args.brew_path, "services", "stop", args.name]
+    rc, stdout, stderr = module.run_command(stop_cmd, check_rc=True)
+
+    _exit_with_state(args, module, changed=True)
+
+
+def restart_service(args, module):
+    # type: (HomebrewServiceArgs, AnsibleModule) -> None
+    """Restart the requested brew service. This always results in a change."""
+    if module.check_mode:
+        _exit_with_state(
+            args, module, changed=True, message="Service would be restarted"
+        )
+
+    restart_cmd = [args.brew_path, "services", "restart", args.name]
+    rc, stdout, stderr = module.run_command(restart_cmd, check_rc=True)
+
+    _exit_with_state(args, module, changed=True)
+
+
+def main():
+    module = AnsibleModule(
+        argument_spec=dict(
+            name=dict(
+                aliases=["formula"],
+                required=True,
+                type="str",
+            ),
+            state=dict(
+                choices=["present", "absent", "restarted"],
+                default="present",
+            ),
+            path=dict(
+                default=[
+                    "/usr/local/bin",
+                    "/opt/homebrew/bin",
+                    "/home/linuxbrew/.linuxbrew/bin",
+                ],
+                type="list",
+                element="path",
```

Comment:
@felixfontein is recommending keep the "odd" use of `path` for the time being. Maybe later we can go back to this and fix it across all the `homebrew_*` modules.
Anyways, just for the educational sake, the sanity tests are failing because:
1. it is `elements`, not `element`, and
2. the documentation block by the beginning of the file should match the `argument_spec`

Reply:
D'oh